modules/system/assets/ui/less/flag.less

```diff
@@ -2,280 +2,48 @@
 // Dependencies
 // --------------------------------------------------
 
-@import "global.less";
```

Comment:
Because nothing from global.less is used in this mixin. Everything is taken from the - '@import "../vendor/flag-icon/less/variables";'

Basically, what I did is made css icons (flag-icon-css) usable and generated based on the OctoberCMS standard :)